### ISSUE TITLE
tls 1.2 only server - protocol security negotiation or connection failure

### ISSUE THREAD

**Issue by imp1sh** [FreeRDP/FreeRDP#2996]
Hi,

I'M using version 1.2.5-dev git 13f710b and trying to connect to a RDP Server that supports TLS 1.2 only for security reasons.

Since we disabled TLS 1.0 and 1.1 I cannot connect any more.

This is how I connect:
```bash
/usr/local/bin/xfreerdp /sec:tls /f /v:<<IP>> /u:$(zenity \
--entry \
--title="Domain Username" \
--text="Enter your Username") /p:$(zenity \
--entry \
--title="Domain Password" \
--text="Enter your _password:" \
--hide-text) 
```

Also with /sec:tls it doesn't work.

This is the message I'm getting:
```
[14:44:58:158] [10149:10150] [WARN][com.freerdp.core.nego] - Error: HYBRID_REQUIRED_BY_SERVER
[14:44:58:158] [10149:10150] [ERROR][com.freerdp.core.nego] - Protocol Security Negotiation Failure
[14:44:58:158] [10149:10150] [ERROR][com.freerdp.core] - freerdp_set_last_error ERRCONNECT_SECURITY_NEGO_CONNECT_FAILED [0x2000C]
[14:44:58:158] [10149:10150] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure
[14:44:58:158] [10149:10150] [ERROR][com.freerdp.client.x11] - Authentication only, exit status 1
```

**Comment by lemonmojo** [comment]
@imp1sh See https://github.com/FreeRDP/FreeRDP/issues/2828

**Comment by bmiklautz** [comment]
Fixed with PR #3046.
